### PR TITLE
Fix case of problem types

### PR DESCRIPTION
**Pull request by mgrabovsky**
Problem types should be lowercase as defined in Satyr.

### CODE REVIEW

File: src/pyfaf/storage/report.py [hunk 1]
```diff
@@ -175,9 +175,9 @@ def frames(self):
     def normalized(self):
```
Comment (mkutlak) [inline src/pyfaf/storage/report.py]: You can remove the function completely as it isnt used anywhere and `btp_thread()` was removed in commit b79e74351a85ccbabde565f2397d36b4629f0b00

File: src/pyfaf/utils/storage.py [hunk 2]
```diff
@@ -19,37 +19,30 @@
...
     """
     Return formatted `reason` of the crash according to report type `rtype`
     """
 
-    if rtype == "USERSPACE":
-        res = RE_SIGNAL.search(reason)
-        if res:
-            return "{0} in {1}".format(res.group(), function_name)
-
+    if rtype == "userspace":
```
Comment (mkutlak) [inline src/pyfaf/utils/storage.py]: I think this should be `core`.

https://github.com/abrt/faf/blob/c7927635e89189dd70bf840498121f6f7c6ab2f4/src/pyfaf/ureport_compat.py#L35

Comment (mgrabovsky) [reply src/pyfaf/utils/storage.py]: Yup, thanks.

**Comment by mkutlak** [comment]
LGTM :+1:

**Pull request merged**